### PR TITLE
RMMIS-5820-pre-populate-fields

### PR DESCRIPTION
**Pull request by kjschulz**
RMMIS-5820: Wrote the logic to pre-populate field values from previous page values and/or perform operations on them, wrote tests for both the Field component & the utils, ES6ified ALLTHETHINGS

### CODE REVIEW

**Comment by traviscooperscdhhs** [comment]
:+1:

File: src/Field.jsx [hunk 1]
```diff
@@ -53,15 +54,35 @@ module.exports = React.createClass({
...
+      /**
+       * Provides configuration for performing operations on a field dependent on previous
+       * pages' field values.
+       * @param {object} opConfig - Operation config must include an action and an actionType
+       * along with any other relevant information needed to perform the operation
+       * @param {object} action - The name of the operation to peform
+       * @param {object} actionType - Define what kind of object should be returned from
+       * the operation action
+       */
+      if (config.has('appInputOperationConfig')) {
```
Comment (mallenscdhhs) [inline src/Field.jsx]: sorry, should have mentioned this earlier, but can we rename this to `inputOperationConfig`?

Comment (kjschulz) [inline src/Field.jsx]: yep, watcha you wanna call it @mallenscdhhs?

Comment (kjschulz) [inline src/Field.jsx]: @mallenscdhhs, updated.

**Pull request merged**